src/test/shared/codelens/pythonCodeLensProvider.test.ts

```diff
@@ -0,0 +1,76 @@
+/*!
+ * Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+import * as assert from 'assert'
+import { getLocalRootVariants } from '../../../shared/codelens/pythonCodeLensProvider'
+
+describe('getLocalRootVariants', async () => {
+    if (process.platform === 'win32') {
```

Comment:
Will you be able to run these tests upstream on a build box?

Reply:
We already have some platform specific tests that run on mac (Travis) and linux (Travis, CodeBuild)

Reply:
Confirmed that Travis doesn't run these tests (see: https://travis-ci.org/aws/aws-toolkit-vscode/jobs/521971775#L1062 ), we may add Windows tests at a later point: https://blog.travis-ci.com/2018-10-11-windows-early-release